src/styles/mixins/_buttons.scss

```diff
@@ -0,0 +1,8 @@
+@mixin bdl-Button-margins($margin) {
+    margin: $margin;
+
+    &:hover,
```

Comment:
and :focus?

Reply:
I didn't see a margin for :focus, but I might be missing it? The rule I saw was `.btn-plain, .btn-plain:hover, .btn-plain:active` in _buttons.scss

Reply:
Got it. https://github.com/box/box-ui-elements/blob/master/src/styles/common/_buttons.scss

What a weird set up.